src/screens/TopicPageScreen/elements/NavHeader.js

```diff
@@ -156,11 +200,31 @@ NavHeader.propTypes = {
   setMemberCount: PropTypes.func,
   memberCount: PropTypes.number,
   topicDetail: PropTypes.object,
-  setIsFollow: PropTypes.func
+  setIsFollow: PropTypes.func,
+
+  searchText: PropTypes.string,
+  setSearchText: PropTypes.func,
+  setDiscoveryLoadingData: PropTypes.func,
+  isFocus: PropTypes.bool,
+  setIsFocus: PropTypes.func,
+  fetchDiscoveryData: PropTypes.func,
+  onCancelToken: PropTypes.func,
+  placeholderText: PropTypes.string,
+  setIsFirstTimeOpen: PropTypes.func,
+  getSearchLayout: PropTypes.func,
+  hasSearch: PropTypes.bool
 };
 
 const styles = StyleSheet.create({
-  Nav: (isHeaderHide, top) => ({
+  container: (animatedHeight) => ({
+    width: '100%',
+    height: animatedHeight,
+    backgroundColor: colors.lightgrey,
+    position: 'absolute',
+    zIndex: 80,
+    overflow: 'hidden'
+  }),
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The style definition for the container seems fine. The dynamic height based on `animatedHeight` should work as expected.



<!-- This is an auto-generated comment by OSS CodeRabbit -->